src/types.md

```diff
@@ -400,6 +471,33 @@ let bo: Binop = add;
 x = bo(5,7);
 ```
 
+### Other traits
+
+Closure types implement the following traits, if allowed to do so by the
```

Comment:
The actual values captured don't matter, but the types of the variables do. I'd personally write it "Closures implement the following traits conditionally on the types of the captured variables."

Reply:
That's a fair point. But it also doesn't necessarily depend on the type of the variable captured, but the type of the capturing variable in the closure. For instance, capturing a non-`Copy` type by reference is fine for `Copy`, since the reference is itself `Copy`.

Reply:
Found a way to make this work, please take a look once I push.